### ISSUE TITLE
Facility Operating Status - Expandable Alerts - UX review and revisions

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18225]
## Description
Recently [an accessibility issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2839) with the expandable alert - error variant (red) and the color contrast on the hover state was reported. After doing so, our team was informed that this particular variant of the component had actually been removed from guidance and is no longer recommended. 

We should review our use cases across Facilities including user flows to determine if there is a better way to present this information to users.

Do expandable alerts still make sense? Should a different variant or component be considered? Informational information may still make sense with the expandable component but creates user friction when important information is hidden behind an extra click in an expandable alert.

---
**Edit by Michelle 9.27.2024**
- this work should be done in tandem with the expansion of facility operating status to support 2-3 additional statuses, including "Closing" and "Coming soon" so that decisions can be made about colors and icons to represent the full range of statuses [#13598 Iterate on Facility Operating Status](https://app.zenhub.com/workspaces/vagov-sitewide-product-team-6645154188a81a1f7a0a9464/issues/gh/department-of-veterans-affairs/va.gov-cms/13598)
   - As a Veteran, I want to know if a new facility is opening near me and want to know if it is open and delivering services so that I can plan to visit a location that is available. 
   - As a Veteran, I want to know in advance that a location near me will not be available in the future so that I can plan to visit a location that is available. 

- expandable alerts are currently the only way on Vet Center and Regional Office pages to present the critical "more info"/details which explain the status 

 ---

## User stories

As a Veteran I am able to see important information so that I don't miss vital details needed for my upcoming visit to my facility.  

As Veteran I want to know my facility's operating status while I am on the facility page so I can quickly understand when I can go and not taken away to a different page. 

## Artifacts created
- [Working Figma file](https://www.figma.com/design/HcEIBH2fxd147gLK8u2zgt/Facility-Operating-Status?node-id=1-6820&t=FbDYARRAS5vSlAmJ-1)

## Acceptance Criteria
- [ ] Use cases and user flows are reviewed and revised for optimal UX
- [ ] Design revisions in Figma - Veteran facing
- [ ] Review for EX, if CMS changes are needed, create spin off ticket
- [ ] Requires dev review
- [ ] Requires design review
- [ ] Requires accessibility review
- [ ] Implementation ticket(s) created

**Comment by aklausmeier** [comment]
@laflannery @mmiddaugh revised/refined but will need team's refinement review when it comes time to pull this in

**Comment by jilladams** [comment]
Tagging this into VBA National Epic since it's currently orphaned and gonna be easy to lose and that's the next logical batch of things we'll consider for an iteration.

**Comment by Agile6MSkinner** [comment]
@davidmpickett @thejordanwood @aklausmeier Let's talk about this one in UX sync this week.

**Comment by Agile6MSkinner** [comment]
Move epics from VBA ROs National Rollout.

**Comment by jilladams** [comment]
Noting Amanda as assignee here, since she's been working it.

**Comment by aklausmeier** [comment]
We might be able to call this one complete and closed unless the team needs revisions.

**Comment by jilladams** [comment]
I'd like us to keep it open until we've created the implementation tickets for it. Too easy to lose that work otherwise.
